### ISSUE TITLE
Crashes at launch on macOS

### ISSUE THREAD

**Issue by niik** [desktop/desktop#16052]
**TL;DR: If you're using macOS and GitHub Desktop is crashing on launch please manually [download the latest version](https://desktop.github.com/) and replace your installed version with it.**

GitHub Desktop versions 3.0.3 - 3.1.3 will fail to launch on macOS due to a [revoked code signing certificate](https://github.blog/2023-01-30-action-needed-for-github-desktop-and-atom-users/) starting on Thursday February 2nd 2023.

If you're encountering a crash dialog when launching GitHub Desktop please download the latest version of GitHub Desktop from [desktop.github.com](https://desktop.github.com/) and use it to replace your existing app. This will not affect your preferences or tracked repositories and you'll be able to carry on as soon as you've installed the new version.

# What does it look like

The exact message differs between macOS versions but on a recent macOS installation it might look like this
| Crash dialog | Details |
|--------|--------|
| <img width="279" alt="macOS dialog stating that GitHub Desktop quit unexpectedly" src="https://user-images.githubusercontent.com/634063/217194701-aab7977a-76c9-48dc-bd7b-60b6ab7b0555.png"> | <img width="300" alt="macOS dialog showing detailed information about why GitHub Desktop crashed unexpectedly. Within the details it says that the crash is due to the code signature being invalid" src="https://user-images.githubusercontent.com/634063/217195064-9e0a80b4-7e7b-4c3d-9ec0-45ba345cb066.png"> | 

# How do I replace my current version?

Download the latest version of GitHub Desktop from https://desktop.github.com/

1. Locate the downloaded application in your downloads folder
1. Depending on which browser you use and your preferences the downloaded application may be a zip file which you’ll first have to unpack. Typically double clicking on the zip file in Finder will suffice
1. Now drag the GitHub Desktop application from your downloads folder to the Applications folder
1. Approve the dialog asking if you want to replace the existing application
1. Launch GitHub Desktop, you should see a prompt saying that GitHub Desktop was downloaded from the Internet, approve this dialog as well
1. You’re all done!

**Comment by niik** [comment]
We expect that this will affect a fair number of users so in order to reduce noise and +1-type comments in this issue I'm going to lock it. If the instructions above doesn't work for you please see if there's another open issue which describes your scenario or create a new one and we'll work to incorporate information from those issues into this one.

**Comment by niik** [comment]
Going to close this now that we've shipped updates that have been available for a good while
